include/rocksdb/options.h

```diff
@@ -1356,6 +1356,13 @@ struct ReadOptions {
   // processing a batch
   std::chrono::microseconds deadline;
 
+  // It limits the maximum cumulative value size of the keys in batch while
+  // reading through MultiGet. Once the cumulative value size exceeds this
```

Comment:
The comment can probably be reworded a little to match the behavior - "cumulative size is not allowed to exceed the limit".

Reply:
The comment is still a little confusing as it may be interpreted that the cumulative value size of successful keys can exceed ```value_size_hard_limit```. Maybe we can say "Once a key causes the cumulative size to exceed this hard limit, that key and all remaining keys are returned with status Aborted.".